instant-xml/tests/all.rs

```diff
@@ -61,7 +62,7 @@ fn struct_with_custom_field() {
         }
         .to_xml()
         .unwrap(),
-        "<StructWithCustomField xmlns=\"URI\" xmlns:bar=\"BAZ\" xmlns:foo=\"BAR\"><Nested><bar:flag>true</bar:flag></Nested></StructWithCustomField>"
+        "<StructWithCustomField xmlns=\"URI\" xmlns:bar=\"BAZ\" xmlns:foo=\"BAR\"><Nested xmlns=\"URI\" xmlns:bar=\"BAZ\"><bar:flag>true</bar:flag></Nested></StructWithCustomField>"
```

Comment:
In this case we should not repeat the `xmlns=` and `xmlns:bar=` attributes for the `Nested` tag, because they are implied by the parent tag.

If you like, we can save this for the future since this serialization is correct, but in that case I'd like you to file an issue for it so what we don't forget.

Reply:
You are right, this PR name is a little misleading, as it evolved from simple-deserializer so it is touching only the deserializer part. I'm currently working on the Serializer part, I was thinking of separating it into other PR, are you ok with it? This change was just the convenience for the test to pass as `Nested` struct is used in some other tests, I can separate better though.

Reply:
Sure, it's fine to separate it into a separate PR.

Reply:
I separated the serializer test and will open the serializer PR soon.